lib/Migrator/FilesMigrator.php

```diff
@@ -147,7 +147,7 @@ private function estimateFolderSize(Folder $folder, ?callable $nodeFilter = null
 				$size += $this->estimateFolderSize($node, $nodeFilter);
 			}
 		}
-		return $size;
+		return (int)$size;
```

Comment:
Will fail on 32bits for more than 4GB (or 2GB because signed?).
Should we move all int to int|float for filesizes 32bit support?

Reply:
@Pytal ^

Reply:
🤔 yep 2GB max cause signed https://www.php.net/manual/en/reserved.constants.php#reserved.constants.core

32-bit support added in:
- https://github.com/nextcloud/server/pull/38104
- https://github.com/nextcloud/user_migration/pull/408